### PR TITLE
Removing % sign from telegram message as it is already a pct.

### PR DESCRIPTION
**Pull request by mishaker**

### CODE REVIEW

File: freqtrade/rpc/telegram.py [hunk 1]
```diff
@@ -208,12 +208,12 @@ def _status(self, bot: Bot, update: Update) -> None:
...
                     # Adding initial stoploss only if it is different from stoploss
                     "*Initial Stoploss:* `{initial_stop_loss:.8f}` " +
-                    ("`({initial_stop_loss_pct:.2f}%)`" if r['initial_stop_loss_pct'] else "")
+                    ("`({initial_stop_loss_pct:.2f})`" if r['initial_stop_loss_pct'] else "")
                     if r['stop_loss'] != r['initial_stop_loss'] else "",
 
                     # Adding stoploss and stoploss percentage only if it is not None
                     "*Stoploss:* `{stop_loss:.8f}` " +
-                    ("`({stop_loss_pct:.2f}%)`" if r['stop_loss_pct'] else ""),
+                    ("`({stop_loss_pct:.2f})`" if r['stop_loss_pct'] else ""),
```
Comment (xmatthias) [inline freqtrade/rpc/telegram.py]: excuse my ignorance - but where does it have the % from?

The column is numeric, and in `rpc.py` i don't see an added % sign ... ?

https://github.com/freqtrade/freqtrade/blob/dfed7136475f5b164b7e9e6eefa28dd53c3e45f3/freqtrade/rpc/rpc.py#L103-L119

Comment (mishaker) [reply freqtrade/rpc/telegram.py]: Screen Shot 2019-04-03 at 3 51 45 PM

Comment (mishaker) [reply freqtrade/rpc/telegram.py]: should be either -0.04 or -4%

Comment (mishaker) [reply freqtrade/rpc/telegram.py]: Screen Shot 2019-04-03 at 3 54 41 PM

Comment (xmatthias) [reply freqtrade/rpc/telegram.py]: ah yes ... but i would rather multiply by 100 in that case? 
having 2 numbers (one in parenteses), both without unit is not really clear ... and a stoploss could be 4.3% (which would result in 0.04 in your presentation)

**Pull request merged**